docs/iot/iot-partners/thingspark.md

```diff
@@ -0,0 +1,27 @@
+# Tizen IoT with ThingsPark Cloud
+
+The ThingsPark service is a cloud-based IoT solution that can be connected with various things. 
+
+Tizen developers can easily develop an application to send IoT data to the cloud in real time.
+
+Real-time data monitoring between Tizen service app and ThingsPark cloud is empowered through ThingsPark plugin.
+
+The following diagram shows the interface between Tizen IoT device and ThingsPark cloud :
+
+
+**Figure: Architecture**
+
+![Architecture](media/architecture_thingspark.png)
+
+The ThingsPark plugin can be installed on Tizen IoT devices compatible with as follow:
+
+* Platform : Tizen 5.0 IoT Headed & Headless and later
+* SDK : Tizen Studio 3.3 and later
```

Comment:
- Please use "-" for a bulleted list according to Markdown style.
  For more information, see https://github.com/Samsung/tizen- 
  docs/blob/master/styleguide/style.md#lists.
-  Please change "later" to "higher".
    For more information, see https://developers.google.com/style/word-list#letter-H
- Please remove space before ":".